keymage.js

```diff
@@ -8,8 +8,7 @@
 (function(define, undefined) {
 define(function() {
     var VERSION = '1.1.3';
-    var isOsx = typeof navigator !== 'undefined' &&
-        ~navigator.userAgent.indexOf('Mac OS X');
+    var isOsx = /^mac/i.test(navigator.platform);
```

Comment:
I don't care about old IEs.

1, 2, 3 doesn't really matter, since navigator.platform is deprecated. Plus I like not using regular expressions when you don't need to.

4 is interesting. Not sure if it's worth spending time on speeding up, but maybe it is. :)